### ISSUE TITLE
Feature Request: Add tokens to leaderboard

### ISSUE THREAD

**Issue by corpetty** [AddrETH/addreth#28]
This is possible, but adding to "total" makes it difficult for the following reasons:
- need universal relative value to compare to ( ETH / USD / etc )
- price discovery for tokens is not done through quality oracles, need to integrate multiple APIs to do this
- amount screws up sorting algorithm for leaderboard if you don't care about value

I would say just adding the token to the leader board is start, going to the bottom because it is usually a "zero value transfer"

**Comment by corpetty** [comment]
furthermore, message becomes useless, so it should be turned off, or added through another means outside of the `data` field.

**Comment by geleeroyale** [comment]
Good rundown of the implications associated with introducing tokens. Would love to keep message intact for now. Should we do single token support to ease stuff out (DAI is an obvious first step)?

**Comment by corpetty** [comment]
For extensibility purposes, I think making it as general as possible is the way to go.  Having a UI that added a single token would be a good first step, but the addition is filtered through putting in the ticker "DAI" or whatever token you're looking for.

**Comment by geleeroyale** [comment]
yes! I like that very much - if you have concrete plans for implementation, please go ahead!
